rsocket-core/build.gradle.kts

```diff
@@ -36,6 +36,7 @@ kotlin {
         }
         val commonTest by getting {
             dependencies {
+                implementation("app.cash.turbine:turbine:0.2.1")
```

Comment:
nice!

Reply:
yeah, first try of that library, looks really good!